sass/_base.sass

```diff
@@ -1343,7 +1343,7 @@ $feature-box-div-margin-bottom: 40px
       background-image: url(/images/nav_logo2.svg)
 
   #hero
-    padding-bottom: 20px
```

Comment:
@Abhik1998 Thank you for the fix! ✨ 

But did you verify the other tabs as well, since its a common css? Try to click on Blog/Case studies, I feel there is a padding issue after the header and content below the header. The content looks moved up a bit.

Reply:
Yes I checked but its not padding issue .Moreover the content is correctly placed in blog/ case studies.It wont look good if the content is moved much below